docs/loads/structuralpointdeformation.md

```diff
@@ -0,0 +1,25 @@
+# StructuralPointDeformation
+
+```{warning}
+New object under construction.
+```
+
+**Imposed deformation of a support**
+
+Displacement or rotation can be imposed on a support. Displacement can be set in the direction of one of the global axes, positive displacement acts in the positive direction of the corresponding axis. Rotation can be imposed around one of the global axes, positive rotation acts right-handed about the corresponding positive axis. A rigid support is required in the direction of the deformation.
+
+```{image} ../.gitbook/assets/41b_deformation_of_support.png
+:width: 600px
+```
+
+## Specification in the excel
+
+| Column header | Data type | Example / enum definition | Required | Description |
+|---|---|---|---|---|
+| Name | String | RS1 | yes | Human readable unique name of the deformation |
+| Support in a node | String | Sn6 | yes | The name of a support in a node on which the deformation load is applied |
+| Direction | Enum | X<br>Y<br>Z<br>Rx<br>Ry<br>Rz | yes | Specifies the direction of deformation. X, Y, Z are the directions of translation. Rx, Ry, Rz are the directions of rotation |
```

Comment:
Can't this enum just be X, Y or Z?
Because either Translation or Rotation can have a value and depending on which one has a value we'll know if it's a translation or a rotation?
Can't think of another object where we have a similar situation though, can you?

The validation rules would be as follows:
Direction is required
Translation or Rotation is required, but not both at the same time

Reply:
We want only 1 deformation per row in excel. It is true, we could have only X,Y,Z and leave out the Rx,Ry,Rz. But to keep it more natural to structural engineers I would suggest leaving it like this. It is still a subject of discussion, let's decide this after we merge it to latest, so we can invite others to the discussion.